astroquery/hitran/reader.py

```diff
@@ -1,13 +1,151 @@
 from collections import OrderedDict
 from astropy.table import Table
+from astropy import log
+from astropy.config import paths
 import os
+try:
+    import urllib.request as urllib2
+except ImportError:
+    import urllib2
+
 
 DATA_DIR = os.path.join(os.path.dirname(__file__), 'data')
+HITRAN_URL = 'http://hitran.org'
+cache_location = os.path.join(paths.get_cache_dir(), 'astroquery', 'hitran')
+if not os.path.exists(cache_location):
+    os.makedirs(cache_location)
 
 dtype_dict = {'f': 'f', 's':'s', 'd':'i', 'e':'f', 'F':'f', 'A':'s', 'I':'i'}
 fmt_dict = {'f': float, 's':str, 'd':int, 'e':float, 'A': str, 'I':int,
             'F': float}
 
+ISO_INDEX = { 'id':0, 'iso_name':1, 'abundance':2, 'mass':3, 'mol_name':4 }
+
+# Copied from hapi.py code
+ISO = {
```

Comment:
maybe it would be better to load this data from a JSON file?  But it's OK as is for now.